CerebNet/run_prediction.py

```diff
@@ -94,6 +102,19 @@ def setup_options():
 
 
 def main(args):
+    """
+    Main function to run the inference based on the given command line arguments.
```

Comment:
add "cerebelum subsegmentation"

also add a reference to the CerebNet paper (see README)